src/mapping/sanity/two-grid.tsx

```diff
@@ -1,36 +1,30 @@
 import React from 'react';
 
-import ExpertAdvice from '@src/components/features/decoupled-components/expert-advice/expert-advice';
-import FeaturedArticle from '@src/components/features/decoupled-components/featured-article/featured-article';
 import { TwoGrid } from '@src/components/features/decoupled-components/two-grid/two-grid';
-import { toPortableText } from '@src/mapping/sanity/featured-article';
+import { componentMapSanity } from '@src/mapping/sanity/mappings';
 
+function capitalizeFirstLetter(input: string) {
+  return input.charAt(0).toUpperCase() + input.slice(1)
+}
 export const TwoGridSanity = (props: any) => {
```

Comment:
please don't use any, use typings

Reply:
The 2grid component can handle for now 6 components, we are going to have a interface which is very long and all properties are correctly mapped with a interface after the component is mapped.